src/components/feedback/Spinner.tsx

```diff
@@ -2,18 +2,15 @@ import classnames from 'classnames';
 
 import { SpinnerSpokesIcon } from '../icons';
 
-/**
- * @typedef SpinnerProps
- * @prop {'sm'|'md'|'lg'} [size='sm']
- * @prop {'text-light'|'text'|'text-inverted'} [color='text-light']
- */
+type SpinnerProps = {
+  size?: 'sm'|'md'|'lg';
+  color?: 'text-light'|'text'|'text-inverted';
+}
```

Comment:
TS syntax doesn't have a way to indicate the default value. This could still be put in a JSDoc comment though.

Reply:
We have been leaning pretty heavily on the commenting element of JSDoc typing. I'll have to do a little poking around to see what common conventions are for documenting/commenting types.